app/views/admin/cfps/_booths_cfp.html.haml

```diff
@@ -1,12 +1,16 @@
 %dt
-  Start Date
+  Type:
+%dd
+  = @cfp.cfp_type.capitalize
+%dt
+  Start Date:
```

Comment:
Adding the `:` where it was missed is unrelated to the commit _Add cfp type in admin/cfp#show_ Can you please add it in a different commit? So 2 commits :wink:

Reply:
Sure :smile: